docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -833,18 +833,23 @@
 
           - name: Wait for subcloud hosts available if not simplex
             block:
-            - name: Set system type for monitor the deloyment
+            - name: Set system type for monitor the deployment
               set_fact:
                 host_offline_retries: >
                   {{ 60 if hosts_count.stdout | int == 2 else 120 }}
                 host_reconcile_retries: >
                   {{ 120 if  hosts_count.stdout | int == 2 else 240 }}
 
-            - name: Get hosts not online or not available if not simplex
-              shell: >
-                kubectl get hosts -o custom-columns='NAME:.metadata.name,
-                OPERATIONAL:.status.availabilityStatus' -n deployment |
-                awk '$NF !~ /online/ {print}'
+            - debug:
+                msg: >
+                  Retry for {{ host_offline_retries }} seconds until all hosts
+                  can be configured.
+
+            - name: Check hosts cannot be configured by the DM
+              shell: |
+                source /etc/platform/openrc;
+                system host-list --format value |
+                awk '$4 != "locked" && ($6 != "available" && $6 != "online") {print $2}'
```

Comment:
Sorry, I think the logic here is wrong: we may need to check if the $4=="locked" && ($6 != "available" && $6 != "online")